charts/hivemq-platform/tests/configuration/configmap/hivemq_telemetry_test.yaml

```
@@ -0,0 +1,124 @@
+suite: HiveMQ Platform Configuration as ConfigMap - Telemetry tests
+templates:
+  - hivemq-configuration.yml
+  - hivemq-custom-resource.yml
+release:
+  name: test-hivemq-platform
+  namespace: test-hivemq-platform-namespace
+chart:
+  version: 0.0.1
+  appVersion: 1.0.0
```

`chart` options and `hivemq-custom-resource.yml` template are not needed in this test file

```suggestion
templates:
  - hivemq-configuration.yml
release:
  name: test-hivemq-platform
  namespace: test-hivemq-platform-namespace
```